qiskit/transpiler/passmanager_config.py

```diff
@@ -39,6 +39,9 @@ def __init__(
         unitary_synthesis_method="default",
         unitary_synthesis_plugin_config=None,
         target=None,
+        init_method=None,
+        optimization_method=None,
+        optimization_level=None,
```

Comment:
I feel that `optimization_level` here is weird.

Reply:
I'm going to move `optimization_level` as a kwarg on the plugin abstract method for building a stage pass manager. That'll be a more natural fit for what is needed by having it here. I only did this to contain all the plugin options in a single argument

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/8305/commits/d731d0601229db7772d933d496bed78ec5c2eb01